src/index.tsx

```diff
@@ -9,6 +9,8 @@ export interface ITriggerProps<T> {
 type PropertyFunction<T> = () => T;
 
 export interface IReactToPrintProps {
+    /** Optional title for document if saved as file */
+    documentTitle?: string
```

Comment:
Nit: please alphabetize (should come after `copyStyles`)

Reply:
alphabetized in commit e2a7c3d6f855d731c60f3cf621efdf46fbd0ca54